### DISCUSSION TITLE
Moving documents from commonalities-wg to commonalities-repo

### DISCUSSION THREAD

**Discussion by shilpa-padgaonkar** [camaraproject/WorkingGroups#260] [General]
Hello All,

As discussed in the last commonalities meeting, we have managed to merge the PR 244 at the end of last week. 

I have already started moving documents to the commonalities repo. Please do not create any new PRs in the commonalities  working group until I have finished the move. I will provide an update as soon as I am done. 

All new issues and PRs after the move, are expected to be created here https://github.com/camaraproject/Commonalities.

Please subscribe here as well https://lists.camaraproject.org/g/sp-com so that I can add you as reviewers to new PRs.

Also, please take the time to look at PR https://github.com/camaraproject/Commonalities/pull/2 so that you can already let me know if you would prefer some different names for repo directories etc. Most of the documentation is already moved as a part of this PR. 

Thanks all for your kind support.

**Comment by shilpa-padgaonkar** [comment]
Documents have been moved to the new repo.
All open issues have also been transferred to the new repo.
Please feel free to open new issues and PRs under https://github.com/camaraproject/Commonalities.

**Reply by murthygorty** [reply]
Hi @shilpa-padgaonkar, thanks a bunch for taking care of this - looks like you have had a busy couple of days with camara :-)

Now that we have a commonalities family,
1) Can we have release versions for the design-guidelines, document-template and glossary docs? 
2) I hope the current 'main' version is not the latest release as there are a few inconsistencies (in my view) about domain-driven and developer-friendliness :-)

Please let me know if you want me to open a new issue for the above 2 for traceability - happy to do that.
cc: @gmuratk

**Reply by shilpa-padgaonkar** [reply]
@murthygorty:  please see issue https://github.com/camaraproject/Commonalities/issues/32

**Comment by murthygorty** [comment]
super thank you @shilpa-padgaonkar  - you are so kind to raise #32, a better version of my input above :-)  you can close this now. I will examine #32.
